src/main/java/org/takes/rq/RqWithHeader.java

```diff
@@ -78,4 +78,15 @@ public InputStream body() throws IOException {
         );
     }
 
+    /**
+     * Add additional multiply headers.
```

Comment:
@ikhvostenkov We're only adding one header here, so it's better to simply say "Add additional header".